src/projectscene/qml/Audacity/ProjectScene/clipsview/TracksClipsView.qml

```diff
@@ -180,19 +208,30 @@ Rectangle {
             id: mainMouseArea
             anchors.fill: parent
 
+            acceptedButtons: Qt.LeftButton | Qt.RightButton
+
             hoverEnabled: true
 
             onWheel: function(wheelEvent) {
                 timeline.onWheel(wheelEvent.x, wheelEvent.pixelDelta, wheelEvent.angleDelta)
             }
 
             onPressed: function(e) {
-                if (!(e.modifiers & (Qt.ControlModifier | Qt.ShiftModifier))) {
-                    playCursorController.seekToX(e.x)
+                if (e.button === Qt.LeftButton) {
+                    if (!(e.modifiers & (Qt.ControlModifier | Qt.ShiftModifier))) {
+                        playCursorController.seekToX(e.x)
+                    }
+                    selectionController.onPressed(e.x, e.y)
+                    selectionController.resetSelectedClip()
+                    clipsSelection.visible = true
+                } else {
```

Comment:
else if (Qt.RightButton) to be precise :)

Reply:
I see that there is a filter `acceptedButtons:` but who knows what will be there in the future